ops/ansible/playbooks-ccs/rotate_bfd_postgres_user.yml

```diff
@@ -0,0 +1,26 @@
+#ansible-playbook rotate_bfd_postgres_user.yml --extra-vars "rds_database= master_user= master_pw= old_user= db_user= db_user_pw= rds_region="
+- hosts: localhost
+  vars:
+    rds_database: 
+    master_user: 
+    master_pw: 
+    db_user:
+    db_user_pw:
+    rds_region:
+
+  tasks:
+    - name: RDS Gather Facts/Info
+#      rds_instance_info: #requires Ansible 2.9+
+      rds_instance_facts:
+        db_instance_identifier: "{{ rds_database }}"
+
+      register: database_info
+
+    - name: Change Pre-existing Users Password
+      postgresql_user:
+        login_host: "{{ database_info.instances[0].endpoint.address }}"
+        login_user: "{{ database_info.instances[0].master_username }}"
+        login_password: "{{ master_pw }}"
+        db: postgres
+        name: "{{ db_user }}"
```

Comment:
I don't see a password being set. Maybe I'm missing something, but I think we need a `password` field specified.

Reply:
That's weird, I must've accidentally cut too much during refactoring. I added it back.